flexmeasures/api/v3_0/sensors.py

```diff
@@ -258,6 +270,10 @@ def trigger_schedule(  # noqa: C901
         with a target state of charge of 25 kWh at 4.00pm.
         The minimum and maximum soc are set to 10 and 25 kWh, respectively.
         Roundtrip efficiency for use in scheduling is set to 98%.
+        Consumption should be priced by sensor 9,
```

Comment:
The consumption price is being applied to the aggregate consumption of the devices, instead of being applied to the consumption of the single flexible device under consideration.

How about "Aggregate consumption is priced [...]"?

Reply:
Maybe simply adding this (current) assumption which you just spelled out would be most helpful, while also giving some information about the scope:

"Consumption (of all assets in this EMS) should be priced by sensor 9,"